src/components/ItsKevButton.js

```diff
@@ -0,0 +1,11 @@
+import '../css/App.css';
+
+const ItsKevButton = (props) => {
```

Comment:
I know you may need them eventually but you wouldn't usually call out a `props` parameter if you're not using them